### ISSUE TITLE
Update Halant

### ISSUE THREAD

**Issue by frederik-jimdo** [google/fonts#4301]
**Font Project Git Repo URL:**
https://github.com/google/fonts/tree/main/ofl/halant

**Current version:**
v1.101

**New available version:**
v1.102

**Super short description of the upgrade:**
bug fix for misalignment of Latin diacritic glyphs

**Bugs reported in this repo which this upgrade would fix:**
https://github.com/google/fonts/issues/1068

[Halant-v1.102.zip](https://github.com/google/fonts/files/8074109/Halant-v1.102.zip)

**Comment by thlinard** [comment]
This would probably be better:

**Font Project Git Repo URL:**
https://github.com/itfoundry/halant

**Current version:**
v1.101

**New available version:**
v2.000

**Super short description of the upgrade:**
Many improvements.

**Bugs reported in this repo which this upgrade would fix:**
#1068
#105 (partially)

**Comment by frederik-jimdo** [comment]
@thlinard I agree. The new version does have some stylistic changes to some glyphs, e.g. the upper case Q, but probably better to go with the latest upstream.

**Comment by frederik-jimdo** [comment]
Any chance this could be added to a future milestone? Anything I can do to help?
